### PR TITLE
Make tests

### PR DESCRIPTION
**Pull request by letrungkien0210**
@binhfoto  implemented UT for user, I don't know we should merge now or after ?

### CODE REVIEW

File: test/user.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+let user = {
+    username: 'admin',
+    password: 'changeit'
+    };
+
+let jwt = '';
+
+describe('User', () => {
+    before((done) => {
```
Comment (binhfoto) [inline test/user.js]: We should add 'after' function to ensure data always be cleaned.

We should do like this:

cleanData = (done) => {
    User.remove({}, (err) => {
        done();
    })
};

before(cleanData);
after(cleanData);

File: test/user.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+chai.use(chaiHttp);
+
+let user = {
+    username: 'admin',
+    password: 'changeit'
+    };
+
+let jwt = '';
+
+describe('User', () => {
```
Comment (binhfoto) [inline test/user.js]: We should add 'after' function to ensure data always be cleaned.

We should do like this:

cleanData = (done) => {
User.remove({}, (err) => {
done();
})
};

before(cleanData);
after(cleanData);

File: server/config/testing.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
     logging: false,
     db: {
-        url: 'mongodb://localhost/namviet-test'
```
Comment (binhfoto) [inline server/config/testing.js]: Please help me edit url to 'mongodb://localhost:27017/company-test'

**Comment by letrungkien0210** [comment]
@binhfoto Fixed, please review.

**Comment by binhfoto** [comment]
please help me solve the conflict and we can merge now :D

**Comment by letrungkien0210** [comment]
@binhfoto  Fixed conflicted, please review.

**Pull request merged**